### PR TITLE
Bug/met 2419 change namespace in dereferencing service

### PR DESCRIPTION
**Pull request by jochen-vermeulen**

### CODE REVIEW

File: metis-enrichment/metis-enrichment-client/src/main/java/eu/europeana/enrichment/utils/EntityMergeEngine.java [hunk 1]
```diff
@@ -146,6 +148,14 @@ private static AgentType convertAndAddAgent(Agent agent, List<AgentType> destina
     // dateList
     agentType.setDateList(ItemExtractorUtils.extractLabelResources(agent.getDate(), Date::new));
 
+    // placeOfBirth
+    agentType.setPlaceOfBirthList(
```
Comment (acerdas84) [inline metis-enrichment/metis-enrichment-client/src/main/java/eu/europeana/enrichment/utils/EntityMergeEngine.java]: do you have a format standard (in a file that I can import) for the code ?

Comment (jochen-vermeulen) [reply metis-enrichment/metis-enrichment-client/src/main/java/eu/europeana/enrichment/utils/EntityMergeEngine.java]: We're using the google java format style. 
https://github.com/google/styleguide/blob/gh-pages/intellij-java-google-style.xml

**Comment by acerdas84** [comment]
Is the placeOfDeath and placeOfBirth specifically related to this bug ?

File: metis-enrichment/metis-enrichment-client/src/main/java/eu/europeana/enrichment/utils/EntityMergeEngine.java (hunk 1 above)
Comment (acerdas84) [reply metis-enrichment/metis-enrichment-client/src/main/java/eu/europeana/enrichment/utils/EntityMergeEngine.java]: Thanks

**Pull request merged**